lib/galaxy/tool_util/deps/mulled/mulled_build_tool.py

```diff
@@ -20,13 +20,31 @@
     add_single_image_arguments,
     args_to_mull_targets_kwds,
     mull_targets,
+    target_str_to_targets,
 )
-from .util import build_target
 
 if TYPE_CHECKING:
     from galaxy.tool_util.deps.conda_util import CondaTarget
 
 
+def _mulled_build_tool(tool, args):
+    """
+    test verifies that the v2 hashes are the identical with and without using a build in the versions
+    >>> import doctest
+    >>> doctest.ELLIPSIS_MARKER = '-ignore-'
+    >>> import argparse
+    >>> _mulled_build_tool("test/functional/tools/mulled_example_multi_1.xml", argparse.Namespace(dry_run=True, base_image="does-not-matter-here-but-test-is-fast", command="build", verbose=True, involucro_path="./involucro"))  # doctest: +ELLIPSIS
```

Comment:
This shouldn't be a doctest, and we don't want to run this with the general unit tests, just the mulled unit tests please.